### PR TITLE
Document jq / yq usage

### PR DESCRIPTION
**Pull request by robert-heinzmann-logmein**
Document jq usage, related to https://github.com/jonnylangefeld/kubectl-mc/issues/18

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -27,6 +27,95 @@ Run `krew install mc` to install mc. You can then use it via `kubectl mc`.
 
 Run `kubectl mc` for help.
 
+## Using `kubectl mc` in automation with jq and yq
+
+The `kubectl mc` command supports native json any yaml output. This allows for effective usage for automations and inventory run scenarios in multicluster setups using `jq`and `yq`.
```
Comment (devkid) [inline readme.md:32]: Typo `any`

Comment (jonnylangefeld) [inline readme.md:32]:
```suggestion
The `kubectl mc` command supports native json and yaml output. This allows for effective usage for automations and inventory run scenarios in multicluster setups using `jq` and `yq`. 
```

File: readme.md [hunk 2]
```diff
@@ -27,6 +27,95 @@ Run `krew install mc` to install mc. You can then use it via `kubectl mc`.
...
+## Using `kubectl mc` in automation with jq and yq
+
+The `kubectl mc` command supports native json any yaml output. This allows for effective usage for automations and inventory run scenarios in multicluster setups using `jq`and `yq`. 
+
+Note: the `-o output` must be specified for the `kubectl mc` command and not the kubectl argument, as it produces different output. When specified as a kubectl argument, the output is not machine parsable as it does not present a valid json / yaml object.
+
+Wrong usage for automation:
+
+```
+shell$ kubectl mc -r testcluster -- get node --sort-by="{.metadata.creationTimestamp}" -o yaml
```
Comment (jonnylangefeld) [inline readme.md:39]:
```suggestion
$ kubectl mc -r testcluster -- get node --sort-by="{.metadata.creationTimestamp}" -o yaml
```
Just to match other shell script examples in the readme

File: readme.md [hunk 3]
```diff
@@ -27,6 +27,95 @@ Run `krew install mc` to install mc. You can then use it via `kubectl mc`.
...
+
+```
+
+Example yaml output:
+
+```
+shell$ kubectl mc -r testcluster -o yaml -- get node --sort-by="{.metadata.creationTimestamp}"
```
Comment (jonnylangefeld) [inline readme.md:46]:
```suggestion
$ kubectl mc -r testcluster -o yaml -- get node --sort-by="{.metadata.creationTimestamp}"
```

File: readme.md [hunk 4]
```diff
@@ -27,6 +27,95 @@ Run `krew install mc` to install mc. You can then use it via `kubectl mc`.
...
+    kind: Node
+	...
+  - apiVersion: v1
+    kind: Node
+```
+
+Example json output:
+
+```
+shell$ kubectl mc -r testcluster -o json -- get node --sort-by="{.metadata.creationTimestamp}"
```
Comment (jonnylangefeld) [inline readme.md:68]:
```suggestion
$ kubectl mc -r testcluster -o json -- get node --sort-by="{.metadata.creationTimestamp}"
```

**Review by jonnylangefeld — APPROVED** [review]
Awesome, thank you so much for the contribution!
I think this is really helpful.
I might use the `# UX` topic above the `jq and yq` topic, just so that someone who only takes a glance at the readme gets a feel for how it can be used. And then they can scroll a little further to see how it can be used with jq and yq.

**Pull request merged**